run-tests.sh

```diff
@@ -76,19 +76,14 @@ SCREENSHOTS_DIR="${CUCUMBER_DIR}/screenshots"
 export ROOT_URL="${HOST}:${PORT}"
 
 cd ${TESTS_DIR}
-source ~/.nvm/nvm.sh
+source ${NVM_DIR}/nvm.sh
 nvm use ${NODE_VERSION}
 npm ci
 
-# chimp installs the latest version of chromeDriver which does not work, hence the below.
-cd node_modules/chimp
-npm install chromedriver@2.35
-cd -
-
 rm -rf ${SCREENSHOTS_DIR}
 
 # Hotfix: change node debug option in cucumber
-sed -ie 's/--debug/--inspect/g'  ${TESTS_DIR}/node_modules/chimp/dist/lib/cucumberjs/cucumber.js
+# sed -ie 's/--debug/--inspect/g'  ${TESTS_DIR}/node_modules/chimp/dist/lib/cucumberjs/cucumber.js
```

Comment:
We can keep this commented out, maybe let's add a note that this command is to enable debugging based on the WebStorm/Pycharm setup in https://www.jetbrains.com/help/webstorm/running-and-debugging-node-js.html#node_debugging_overview

Reply:
Added a description comment for why this is useful.